### DISCUSSION TITLE
Display rate and DA settings on score panels without tooltip

### DISCUSSION THREAD

**Discussion by apollo-dw** [ppy/osu#20799] [Ideas] [answered]
I believe that non-standard HT/DT rates and Difficulty Adjust settings are going to see a widespread use once more people start to use Lazer competitively (going for map leaderboard rank 1s or player performance). Because of this, I believe that rate-changing mods and DA should recieve special treatment in the display of a score on the leaderboard.

This would, in my opinion, improve UX and the time it takes for a user to understand a score based on looking at it. Currently, this requires user interaction: hovering over the mod icon (example shown below).

![image](https://user-images.githubusercontent.com/83023433/196167796-3f728440-e2e6-4ce9-94c6-6b49a5a23840.png)

Currently, there isn't even anything signaling that this score uses something other than the default 1.5x rate for DT, which is no good in the context of seeing 6 or so scores at once on the leaderboard. The song select drafts on figma presently don't appear to accomodate for this.

This could be done is displaying settings within the icon, but would require good execution for it not to be abhorrently ugly. In the current song select draft, perhaps the "DT" mod text could literally just turn into "1.3x". That seems naive though.

**Comment by integerrr** [comment, marked as answer]
#19750?

**Reply by apollo-dw** [reply]
Yeah there it is... I tried to search for something like this but I couldn't find it. This discussion definitely addresses the "how do I know this has mod settings" point. I hope that it's not lost that I am specifically suggesting that rate-changing mods and (possibly) DA deserve special treatment though.

**Reply by integerrr** [reply]
Just curious, how do you envision the "special treatment" to look?
